src/open_inwoner/scss/components/Form/FileInput.scss

```diff
@@ -49,12 +64,42 @@
     text-align: center;
   }
 
+  .p--upload-error {
```

Comment:
Modifiers should have names that make sense in a very generic way. "upload-error" is not a generic style for a piece of text. 

- If the styling is very specific to a component than the "upload-error" should be a part of that component (and the classname should look something like `file-input__upload-error`.
- If however, the styling a generic way to demonstrate errors (or "danger texts") `p--danger` might be better here (but then it should be defined in the scss file for the "P" component.
- If we need to reuse the "upload error" and it's to complex to be composed out of other components with modifiers it can be a block on it's own. In which case it should have it's own file/folder in at least in the scss structure. 

Based on the code my guess is that first scenario applies here. 

BEM in short:

- Block: Component name for something reusable, e.g. `card`.
- Element: A part of a component that cannot be (re)used without a block, e.g. `card__body`.
- Modifier: An additional class that indicates a certain "state", "theme", "type" etc. of either a block or an element, e.g. `card card--info` or `card__body card__body--direction-vertical`.